src/main/java/uk/gov/ida/eidas/cli/trustanchor/Print.java

```diff
@@ -25,20 +31,36 @@
     @Option(names={ "-o", "--output" }, description="File to output to. Defaults to stdout.", required=false)
     private File outputFile;
 
+    private CountryTrustAnchorValidator validator = CountryTrustAnchorValidator.build();
+
     @Override
     public Void call() throws Exception {
         final int INDENT_FACTOR = 4;
         JSONArray anchorObjects = new JSONArray();
+        List<String> invalidFiles = new ArrayList<String>();
 
         for (File anchor : anchors) {
             String encodedJwsObject = FileReader.readFileContent(anchor);
-            anchorObjects.put(JWSObject.parse(encodedJwsObject).getPayload().toJSONObject());
+            JSONObject jws = JWSObject.parse(encodedJwsObject).getPayload().toJSONObject();
+            List<JWK> keys = JWKSet.parse(jws).getKeys();
```

Comment:
How many keys would one anchor have? I'm a little confused about the terminology I think.

Reply:
Yeah, we incosistently use the term anchor for different things. Essentially, each aggregated JWS file is composed of multiple JWK "keys", of which each has one associated public key (or a certificate chain).

Reply:
Would you say the trust anchor is the aggregated JWS file, containing the key for each country? Or is each key in that file an anchor on its own?